tasks/unix-shell.md

```diff
@@ -0,0 +1,79 @@
+   Windows `cmd` or PowerShell. You must have already installed `git`
+   from [git-scm.com](https://git-scm.com/downloads).
+   It offers Git Bash that can be used to type bash commands in.
+
+   Right click anywhere in file explorer and select
+   `Git Bash here` option.
+
+   ![windows-git-bash](../img/windows-git-bash.png)
+
+2. Pass all Learning the Shell section on [linuxcommand.org](http://linuxcommand.org/).
+
+<!-- [Web Development](https://www.udacity.com/course/web-development--cs253) -->
+<!-- [Designing RESTful APIs](https://www.udacity.com/course/designing-restful-apis--ud388) -->
+
+Finish the course (1) and also read articles (2) and (3) above.
+
+When complete do the following:
+
+1. Make a screenshot of your finished lessons from (1)
+   and put the screenshot in `task_unix_shell` folder in
+   your `kottans-backend` repo
+1. In your `kottans-backend` repo `README.md`:
+   - add header `## Unix Shell`
+   - embed the screenshot or add a link to the screenshot
+   - list your reflections on all subtasks
+     (_what was new to you_, _what surprised you_, _what you intend to use in future_)
+1. If you honestly finished all the previous steps then go ahead
+   and share it with others –
+   post a message in [course channel][chat]:
+   `Unix Shell — #done` (or `Unix Shell — #p2p_done` if you are p2p course student) and add the link to your repo. **This step is important, as it helps mentors to track your progress!**
+1. Study Extra Materials below to improve your skills.
+   If you feel it affects your overall course performance consider
+   reverting to those later e.g. when you have all mandatory tasks completed.
+1. You may skip Optional materials from this task (if any).
+
+When you finish this task you can proceed to the next one.
+
+## Extra Materials
+
+- [Linux Bash Shell Cheat Sheet](https://learncodethehardway.org/unix/bash_cheat_sheet.pdf)
+- [Advanced Bash-Scripting Guide. An in-depth exploration of the art of shell scripting](http://www.tldp.org/LDP/abs/html/index.html)
+- [A Guide to 100 (ish) Useful Unix Commands ](http://oliverelliott.org/article/computing/ref_unix/)
+
+## Optional
+
+1. [Command Line Power User](https://commandlinepoweruser.com/)
+
+2. [Configuring Linux Web Servers](https://www.udacity.com/course/configuring-linux-web-servers--ud299)
+
+3. [The curl guide to HTTP requests](https://flaviocopes.com/http-curl/)
+
+## Done?
+
+➡️ Go forward to [Git for Team Collaboration](git-collaboration.md)
+
+⤴️ Back to [Contents](../contents.md)
+
+[icon-chat]: https://img.shields.io/badge/chat-on%20telegram-blue.svg
+[icon-mit]: https://img.shields.io/badge/license-MIT-blue.svg
+[icon-awesome]: https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg
+[license]: https://github.com/Kottans/web/blob/master/LICENSE.md
+[awesome]: https://github.com/sindresorhus/awesome#front-end-development
+[chat]: https://t.me/joinchat/CX8EF1JmLm9IM6J6oy2U7Q
```

Comment:
Are you sure we need the link to the FE chat here?

Reply:
@suchov its magically gone :)